.github/workflows/build-publish-jars-workflow.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   publish-jar:
     environment: publish-artifacts
     name: Publish JAR to Github package repository
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
I wasn't sure whether the jar that we package should be using 22.04 as node-runners would be using 20.04  as per docs [here](https://docs.radixdlt.com/main/node-and-gateway/node-setup-introduction.html#_prerequisites) 

However I see we previously been using latest. So it may be okay as no one reported issue so far

Reply:
Should we resolve this thread then or do you want to do some extra tests for this action?

Reply:
Yeah should be fine. I will add a test to ticket where I need to do changes for CLI